pkgs/build-support/setup-hooks/multiple-outputs.sh

```diff
@@ -164,6 +169,11 @@ _multioutDevs() {
     done
 }
 
+_multioutStatic() {
+    if [ "$outputs" = "out" ] || [ -z "${moveToStatic-1}" ]; then return; fi;
```

Comment:
Do we want this to run if there is only `[ "out" "lib" ]`? If not, try `[[ "$outputs" = *static* ]]`